src/services/is-dev.ts

```diff
@@ -0,0 +1,2 @@
+export const isDev = process.env.NODE_ENV === 'development'
```

Comment:
this is confusing, since `idDev` implies that it's a dev _config_, not a dev _target_

also, I'd consider doing `process.env.NODE_ENV !== 'production'` 

since that's what's set explicitly for prod targets, whereas for instance, in tests, `process.env.NODE_ENV` is set to `"test"`

Reply:
good point, actually. I'll just ref it directly.